src/lib/utils.ts

```diff
@@ -1,29 +1,17 @@
 
-	for (const letterNumber of lettersNumbers) {
-		result = String.fromCodePoint(codePointLowerA + letterNumber) + result;
-	}
+export function returnToStack(letters:string) {
+	console.log(`Returning: ${letters}`);
+	hint_stack.push(letters)
+}
 
-	return result;
+export function getLettersFromNumber(hintNumber: number): string {
+	let letters = hint_stack.pop() || "whoops"; // TODO:
```

Comment:
We could remove the or here and in hints check if hint text is undefined, and if it is don't add the hint node to the hints container.

Reply:
Like I said, the chances of running out of hints are minimal